HW3/Menues/Database/DataBaseMenu.cs

```diff
@@ -47,18 +47,19 @@ protected override void SelectMenuItem()
             ShowCommands();
         }
 
-        public void ShowCommands()
+        private void ShowCommands()
```

Comment:
lines 22-26
may be put it into 'finally'?

Reply:
What is 'default'?

Reply:
i'd rather leave check before switch